diagnostic.md

```diff
@@ -9,6 +9,8 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+boolean = true,number = 1string = 'Hi',Null = null,Undefined = y is 7
+
```

Comment:
Confused by your example for `undefined`. Any declared but yet-to-be-defined variable is undefined, so I'm assuming you meant something along those lines? Happy to chat if undefined is unclear at all!